### DISCUSSION TITLE
Welcome to ECIPs Discussions!

### DISCUSSION THREAD

**Discussion by q9f** [ethereumclassic/ECIPs#436] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gitr0n1n** [comment]
>To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hi, I'm r0n1n. I like my networks like I like my hunnies- resilient. I volunteeer time to ETC where I find interest.
(I always edit my messages because I have a lot of typos.)

**Comment by justjinn** [comment]
Hi, 


I'm justjinn. I'm here to learn as much as I can about Ethereum Classic and the crypto ecosystem at large, support the ETC ecosystem in any way possible for me, collaborate with community member on projects where I qualify, and to meet positive individuals to past the time with. :)
